x-pack/test/apm_api_integration/tests/dependencies/top_dependencies.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { APIReturnType } from '../../../../plugins/apm/public/services/rest/createCallApmApi';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { registry } from '../../common/registry';
+import { dataConfig, generateData } from './generate_data';
+import { NodeType, BackendNode } from '../../../../plugins/apm/common/connections';
+import { roundNumber } from '../../utils';
+
+type TopDependencies = APIReturnType<'GET /internal/apm/backends/top_backends'>;
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const apmApiClient = getService('apmApiClient');
+  const synthtraceEsClient = getService('synthtraceEsClient');
+
+  const start = new Date('2021-01-01T00:00:00.000Z').getTime();
+  const end = new Date('2021-01-01T00:15:00.000Z').getTime() - 1;
+  const backendName = 'elasticsearch';
+
+  async function callApi() {
+    return await apmApiClient.readUser({
+      endpoint: 'GET /internal/apm/backends/top_backends',
+      params: {
+        query: {
+          start: new Date(start).toISOString(),
+          end: new Date(end).toISOString(),
+          environment: 'ENVIRONMENT_ALL',
+          kuery: '',
+          numBuckets: 20,
+          offset: '',
+        },
+      },
+    });
+  }
+
+  registry.when(
+    'Top dependencies when data is not loaded',
+    { config: 'basic', archives: [] },
+    () => {
+      it('handles empty state', async () => {
+        const { status, body } = await callApi();
+        expect(status).to.be(200);
+        expect(body.backends).to.empty();
+      });
+    }
+  );
+
+  registry.when(
+    'Top dependencies',
+    { config: 'basic', archives: ['apm_mappings_only_8.0.0'] },
+    () => {
+      describe('when data is loaded', () => {
```

Comment:
nit: "data is loaded"  is left-over from we had archives. Perhaps:

```suggestion
      describe('when data is generated', () => {
```

or

```suggestion
      describe('when trace data is ingested', () => {
```

?

Reply:
Makes more sense, thanks